src/api/alchemy-websocket-provider.ts

```diff
@@ -687,22 +691,16 @@ export class AlchemyWebSocketProvider
   /** @internal */
   private customStartEvent(event: EthersEvent): void {
     if (event.type === 'alchemy') {
-      const { address } = event;
-      if (!!address) {
-        void this._subscribe(
-          event.tag,
-          ['alchemy_filteredNewFullPendingTransactions', { address }],
-          this.emitProcessFn(event),
-          event
-        );
-      } else {
-        void this._subscribe(
-          event.tag,
-          ['alchemy_newFullPendingTransactions'],
-          this.emitProcessFn(event),
-          event
-        );
-      }
+      const { fromAddress, toAddress, hashesOnly } = event;
+      void this._subscribe(
+        event.tag,
+        [
+          'alchemy_pendingTransactions',
```

Comment:
Am I reading this correctly that we use the type "alchemy" internally to represent Alchemy pending transaction events? That feels pretty unintuitive, and also confusing if we add other Alchemy-specific streams in the future. The event type should reference transactions somehow.

Reply:
Good idea! changed from `alchemy` to `alchemy-pending-transactions`. Need to use lower case since ethers turns everything lowercase.